test/query/spec.test.ts

```diff
@@ -232,6 +233,53 @@ describe('query/spec', () => {
       });
     });
 
+    it('should produce correct SpecQuery with sort: SortOrder', () => {
+      const specQ = fromSpec({
+        data: {values: [{x: 1}, {x: 2}]},
+        transform: {filter: 'datum.x ===2'},
+        mark: Mark.POINT,
+        encoding: {
+          x: {field: 'x', sort: SortOrder.ASCENDING, type: Type.QUANTITATIVE},
+          y: {field: 'x', type: Type.QUANTITATIVE, scale: null}
```

Comment:
Y isn't relevant to this test.  You can either combine two test cases and use different type of sort in x and y or just put one channel.  

I prefer the former.